DigitalLearningSolutions.Web.Tests/Controllers/Login/LoginControllerTests.cs

```diff
@@ -144,5 +144,24 @@ public void Unapproved_delegate_account_redirect_to_not_approved_page()
             // Then
             result.Should().BeViewResult().WithViewName("AccountNotApproved");
         }
+
+        [Test]
+        public void Log_in_with_approved_delegate_id_fetches_associated_admin_user()
```

Comment:
Previously, there was a test to ensure that getting an admin user would fetch the associated delegate users. Since the logic for getting the user is now in the user service, it is obscured from the controller and so can't be tested here. In order to test it in UserServiceTests, the UserService would have to be refactored  so it also has a DataService. I know that Alex has done that refactoring in 393, so when that is all merged I will have to remake that test.

Reply:
Sounds good - do you mind coordinating with him and maybe just making a commit on his branch?